app/src/main/java/me/saket/dank/ui/UrlRouter.java

```diff
@@ -125,13 +128,12 @@ public Intent intent(Context context) {
           DefaultWebBrowser defaultWebBrowser = defaultWebBrowserPref.get();
 
           // Smart linking.
-          if (defaultWebBrowser == DefaultWebBrowser.DANK_INTERNAL_BROWSER) {
-            String packageNameForDeepLink = findAllowedPackageNameForDeepLink(url);
-            if (packageNameForDeepLink != null && isPackageNameInstalledAndEnabled(context, packageNameForDeepLink)) {
-              return Intents.createForOpeningUrl(url).setPackage(packageNameForDeepLink);
-            }
+          @Nullable Deeplink deeplink = findAllowedPackageNameForDeepLink(url);
```

Comment:
`@Nullable` is already specified where the function is defined and shouldn't be necessary here.